src/dodal/devices/areadetector/adaravis.py

```diff
@@ -1,101 +0,0 @@
-from collections.abc import Mapping
-from typing import Any
-
-from ophyd import EpicsSignal, Signal
-from ophyd.areadetector.base import ADComponent as Cpt
-from ophyd.areadetector.detectors import DetectorBase
-
-from .adutils import Hdf5Writer, SingleTriggerV33, SynchronisedAdDriverBase
-
-_ACQUIRE_BUFFER_PERIOD = 0.2
-
-
-class AdAravisDetector(SingleTriggerV33, DetectorBase):
-    cam = Cpt(SynchronisedAdDriverBase, suffix="DET:")
-    hdf = Cpt(
-        Hdf5Writer,
-        suffix="HDF5:",
-        root="",
-        write_path_template="",
-    )
-    _priming_settings: Mapping[Signal, Any]
-
-    def __init__(self, *args, **kwargs) -> None:
-        super().__init__(*args, **kwargs)
-        self.hdf.kind = "normal"
```

Comment:
why is `AdAravis` related?

Reply:
The AdSim was a final remaining "used" ophyd device in this module, so with it being deleted the whole module may as well be removed.